app/assets/stylesheets/modules/icons.scss

```diff
@@ -40,3 +40,16 @@
 .sul-i-remove-2 {
   color: $sul-unavailabe-icon-color;
 }
+
+.mediation-table {
+  .errored {
+    .request-status {
+      color: $sul-unavailabe-icon-color;
```

Comment:
Is this a typo here or upstream? `s/$sul-unavailabe-icon-color/$sul-unavailable-icon-color`